src/types.d.ts

```diff
@@ -0,0 +1 @@
+declare module "*";
```

Comment:
Why's this necessary?

Reply:
This is necessary for 
```import * as contentHash from 'content-hash'``

Reply:
I'm not an expert but try using `const contentHash = require('content-hash')`. @nicosantangelo may have a better workaround

Reply:
I've changed this line to 
```
const contentHash = require('content-hash')
```
To not need this file.